lib/rules/jsdoc.js

```diff
@@ -209,8 +254,8 @@
  *
  * ### checkReturnTypes
  *
- * Reports discrepancies between the claimed in jsdoc and actual type if
- * both exist (code scan)
+ * Reports discrepancies between the claimed in jsdoc and actual type if both
+ * exist (code scan)
```

Comment:
`Checks for differences between the jsdoc and actual return types if both exist.`